### PR TITLE
Add tests for cover routing

### PR DESCRIPTION
**Pull request by asmorodskyi**
Proof run http://kimball.arch.suse.de/tests/349

### CODE REVIEW

**Comment by asmorodskyi** [comment]
@SergioAtSUSE can please review ?

**Comment by SergioAtSUSE** [comment]
reviewing

File: data/wicked/static_address/ifroute-eth0 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+default  10.0.2.2         -  -
+::/0     fd00:cafe:babe::1  -  -
```
Comment (SergioAtSUSE) [inline data/wicked/static_address/ifroute-eth0]: cafe:babe ? XD

Comment (asmorodskyi) [reply data/wicked/static_address/ifroute-eth0]: :) I just copy/pasted this from here https://github.com/openSUSE/wicked-testsuite/blob/master/test-files/static-addresses/ifcfg-eth0 
but I also notice this :) but as far as I now it is valid IPv6 :)

**Pull request merged**